src/modules/ui/components/DataCard/DataCard.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { Card, Typography } from "antd";
+import { FormattedMessage } from "react-intl";
+import dayjs from "dayjs";
+import { AndroidFilled, AppleFilled, CopyOutlined } from "@ant-design/icons";
+
+import videoImg from "assets/images/thumbnail_vid.png";
+import useFelaEnhanced from "hooks/useFelaEnhanced";
+
+import * as felaRules from "./DataCard.styles";
+import Button from "../Button";
+
+const DataCard = ({ message }) => {
+  const { styles } = useFelaEnhanced(felaRules);
+
+  return (
+    <Card
+      hoverable
+      cover={
+        <img
+          src={message.mediaUploaded ? message.image : videoImg}
+          alt={message.appName}
+        />
+      }
+      className={styles.card}
+    >
+      <div className={styles.contentWrapper}>
+        <div className={styles.cardHeader}>
+          <div>
+            <Typography.Text className={styles.createdAt}>
+              {dayjs(message.date.seconds * 1000).format("DD. MM. YYYY, HH:mm")}
```

Comment:
https://date-fns.org/v2.22.1/docs/format You might want to use this lib, we agreed on using it in the team :))

Reply:
Done ✅